micrometer-java11/build.gradle

```diff
@@ -7,6 +7,7 @@ dependencies {
     testImplementation 'ru.lanwen.wiremock:wiremock-junit5'
     testImplementation 'com.github.tomakehurst:wiremock-jre8-standalone'
     testImplementation project(":micrometer-observation-test")
+    testImplementation libs.awaitility
```

Comment:
This isn't needed, is it?

Reply:
@shakuzen Sorry, you're right. I removed it in d99f984.